Compilation error: unknown type name 'lv_draw_dsc_base_t' in lv_draw_arc.h

## Description
The header `src/draw/lv_draw_arc.h` uses the type `lv_draw_dsc_base_t` (line 30) but does not include `src/draw/lv_draw.h` where this type is defined.

This causes compilation errors when `lv_draw_arc.h` is included in a context where `lv_draw.h` has not already been included (e.g., when compiling `src/core/lv_obj_tree.c` which includes `lv_obj.h` -> `lv_obj_draw.h` -> `lv_draw_arc.h`).

## Steps to Reproduce
1. In an ESP-IDF project with LVGL 9.4.0.
2. Enable `MINIMAL_BUILD` or similar configuration that exposes header dependencies.
3. Compile.

## Logs
```
/managed_components/lvgl__lvgl/src/draw/lv_draw_arc.h:30:3: error: unknown type name 'lv_draw_dsc_base_t'
   30 |   lv_draw_dsc_base_t base;
```

## Suggested Fix
Add `#include "lv_draw.h"` to `src/draw/lv_draw_arc.h`.

--- a/src/draw/lv_draw_arc.h
+++ b/src/draw/lv_draw_arc.h
@@ -13,6 +13,7 @@ extern "C" {
 /*********************
  *      INCLUDES
  *********************/
+#include "lv_draw.h"
 #include "../lv_conf_internal.h"
 #include "../misc/lv_color.h"
 #include "../misc/lv_area.h"

